### PR TITLE
New attempt

### PR DESCRIPTION
**Pull request by rfaber1**

### CODE REVIEW

File: .idea/.gitignore
Comment (MarekWadinger) [inline .idea/.gitignore:1]: Please add .idea folder to .gitignore and remove the files from cache.

Comment (MarekWadinger) [inline .idea/.gitignore:1]: Please add /.idea folder to .gitignore and remove the cached data.

File: GESmodel/CO2.png
Comment (MarekWadinger) [inline GESmodel/CO2.png:1]: Please create a separate folder /results for the output files (.png, .txt, ...)

File: GESmodel/modelFcn.py
Comment (MarekWadinger) [inline GESmodel/modelFcn.py:1]: Please, remove this if not used.

File: GESmodel/simpleModel.py
Comment (MarekWadinger) [inline GESmodel/simpleModel.py:1]: In general, if the state is not used (increment zeroed out in return) it should not be used in the input to the model. If the state is required for computation of other states, than it must be returned if evolves in time. If no evolution is expected, consider converting state to constant.

File: GESmodel/simpleParameters.py
Comment (MarekWadinger) [inline GESmodel/simpleParameters.py:1]: Isn't this a duplicate of core/simpleParameters.py ?

File: core/greenhouse_model.py
Comment (MarekWadinger) [inline core/greenhouse_model.py:1]: Please remove non-informative comments

**Review by MarekWadinger — CHANGES_REQUESTED** [review]
Please address the issues raised in comments on respective changes

**Comment by MarekWadinger** [comment]
@rfaber1  Please revise my proposed changes.

Above that I have a concern regarding computation of CO2 concentration.
Currently, this is considered CO2 concentration:
`    u_co2 = C_c * 1.0e6  # [ppm] >> external C_c_ppm`
However, there is following expression which was used before, and is part of the code:
```
   C_c_ppm = (
        C_c * R * T_i / (M_c * atm) * 1.0e6
    )  # External carbon dioxide concentration [ppm]
```

Could you verify whether current computation is correct?

**Review by rfaber1 — COMMENTED** [review]
ok

**Review by MarekWadinger — APPROVED** [review]

**Pull request merged**